rust/Cargo.toml

```diff
@@ -90,8 +90,9 @@ tracing = "0.1.40"
 [profile.release]
 strip = true
 
-# Full link-time optimization. Reduces binaries by up to 3x on some platforms.
-lto = "fat"
+# Tauri builds are gigantic, "fat" LTO seems to be too much for the CI runners
+# Unfortunately Cargo doesn't allow us to specify thin LTO for Tauri and fat for everything else.
```

Comment:
We can create a custom profile that inherits from release but overrides LTO and build Tauri clients with that profile instead of `--release`.

Reply:
Or you override it with an env var: https://doc.rust-lang.org/cargo/reference/config.html#profilenamelto

Reply:
Further down we override dev profiles for packages? That doesn't work for the GUI clients?

Reply:
Cargo docs say that for some reason `lto` is one of only 3 things that you cannot override https://doc.rust-lang.org/cargo/reference/profiles.html#overrides

![image](https://github.com/user-attachments/assets/242e050b-def8-481d-82b5-4f284f546a66)

I wonder why 🤷‍♀️

Reply:
That is odd, TIL. I guess overriding it with the env var should work? I think we want fat LTO on Linux and MacOS.